docs/MSI/encoder.md

```diff
@@ -50,4 +50,21 @@ That means if D3 = 1 then Y1 Y1 = 11 irrespective of the other inputs. Similarly
 
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/762" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
+## Decimal to BCD Encoder
+{: .no_toc }
+
+The decimal to binary encoder usually consists of 10 input lines and 4 output lines. Each input line corresponds to the each decimal digit and 4 outputs correspond to the BCD code. This encoder accepts the decoded decimal data as an input and encodes it to the BCD output which is available on the output lines. 
```

Comment:
![image](https://user-images.githubusercontent.com/14032427/82402401-e744fd80-9a79-11ea-9ad6-49c39f4efb3c.png)
https://www.geeksforgeeks.org/encoder-in-digital-logic/

Reply:
Is it the plagiarism issue? If that's so, wouldn't it be possible to use geeksforgeeks as a reference site (like tutorialspoint) and mention it in the references section?

Reply:
Made some changes accordingly. Pls review, thanks!